unison-src/transcripts/cycle-update-2.output.md

```diff
@@ -0,0 +1,67 @@
+Update a member of a cycle with a type-preserving update, but sever the cycle.
+
+```unison
+ping : 'Nat
+ping _ = !pong + 1
+
+pong : 'Nat
+pong _ = !ping + 2
+```
+
+```ucm
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These new definitions are ok to `add`:
+    
+      ping : 'Nat
+      pong : 'Nat
+
+```
+```ucm
+.> add
+
+  ⍟ I've added these definitions:
+  
+    ping : 'Nat
+    pong : 'Nat
+
+```
+```unison
+ping : 'Nat
+ping _ = 3
+```
+
+```ucm
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These names already exist. You can `update` them to your
+      new definition:
+    
+      ping : 'Nat
+
+```
+```ucm
+.> update
+
+  ⍟ I've updated these names to your new definition:
+  
+    ping : 'Nat
+    pong : 'Nat
```

Comment:
Is it possible to add a note or indicator on `pong` here to say why/how it was updated? I think this will cause confusion for folks otherwise; "Why is pong updated? I didn't edit pong, did something break?"

Reply:
Yeah, I think we'd prefer `pong` not to appear at all, just like other dependents that we propagate the update to. That's being tracked in the minimegaticket.

Reply:
I think my preferred behaviour would be that `pong` would appear more like an update propagation, since that's _kind of_ what it is; but if it's in the megaticket that's good for now 👍🏼

Reply:
Yeah, that's what I said, or tried to say :)